pkg/jobrunaggregator/jobruntestcaseanalyzer/analyzer.go

```diff
@@ -86,6 +87,21 @@ func (s *testCaseAnalyzerJobGetter) filterJobsForPayload(allJobs []jobrunaggrega
 			(len(s.infrastructure) != 0 && s.infrastructure != getJobInfrastructure(job.JobName)) {
 			continue
 		}
+
+		if s.filters != nil {
+			isFiltered := false
+			for key := range s.filters {
+				if strings.Contains(job.JobName, key) {
+					isFiltered = true
+					break
+				}
+			}
+
+			if isFiltered {
+				continue
```

Comment:
This pattern works fine, but Golang supports named loops to solve this problem as well:

```golang
Outer:
	for i := range allJobs {
		// ...
		if s.filters != nil {
			for key := range s.filters {
				if strings.Contains(job.JobName, key) {
					continue Outer
				}
			}
		}
	}
```

Reply:
That will likely vary by reviewer.  I prefer to see a construct as written here or a helper method with the nested loop to read easier.  Named loops make some people twitch.  I don't recall explicitly writing down something against though.

Reply:
I find them concise and make it much clearer which loop is being continued. 🤷🏻  Anyway, I don't hold up reviews on it, but I think most people don't use it since they don't know it's there.